scipy/sparse/_matrix_io.py

```diff
@@ -106,12 +106,12 @@ def load_npz(file):
     Store sparse array/matrix to disk, and load it again:
 
     >>> import numpy as np
-    >>> import scipy as sp
-    >>> sparse_array = sp.sparse.csc_array([[0, 0, 3], [4, 0, 0]])
-    >>> sparse_array
-    <2x3 sparse array of type '<class 'numpy.int64'>'
-       with 2 stored elements in Compressed Sparse Column format>
-    >>> sparse_array.toarray()
+    >>> import scipy.sparse
+    >>> sparse_matrix = scipy.sparse.csc_matrix(np.array([[0, 0, 3], [4, 0, 0]]))
+    >>> sparse_matrix
+    <2x3 sparse matrix of type '<class 'numpy.int64'>'
+        with 2 stored elements in Compressed Sparse Column format>
+    >>> sparse_matrix.toarray()
```

Comment:
this should all be `array` rather than `matrix` - and whether or not we import scipy `as sp` or not needs to be consistent here and below.

Reply:
This was fixed after resolving the merge conflict. Thank you Lucas.